src/lib/show-plugin-directory.ts

```diff
@@ -13,6 +13,7 @@ export default async function(toolbox: IgniteToolbox) {
   const { print, parameters } = toolbox
   const { colors, newline, info, table, error } = print
   const directory = await fetchPluginRegistry(toolbox)
+  const plugins = installedPlugins(toolbox)
```

Comment:
You should be able to get the plugins by doing something like:

```
const plugins = toolbox.plugins.map(p => p.name)
```

Not in a place where I can test this now, but it should be more accurate than scanning the package.json dependencies.

Give that a shot and let me know how it goes! This is cool!

Reply:
@jamonholmgren I found this list in `toolbox.runtime.plugins` and I updated the PR. Thanks